Makefile

```diff
@@ -60,11 +64,18 @@ test: ## Run the test suite
 format: mix format ## Run formatting tools on the code
 
 
-.PHONY: sbom
+.PHONY: sbom sbom_fast
 sbom: ## creates sbom for both  npm and hex dependancies
 	mix deps.get && mix sbom.cyclonedx -o elixir_bom.xml
-	cd assets/  && npm install && npm install -g @cyclonedx/bom && cyclonedx-bom -o ../bom.xml -a ../elixir_bom.xml && cd ..
-	./cyclonedx-cli convert --input-file bom.xml --output-file bom.json
+	cd assets/  && npm install && npm install -g @cyclonedx/bom@2.0.2 && cyclonedx-bom -o ../$(SBOM_FILE_NAME_CY).xml -a ../elixir_bom.xml && cd ..
+	./cyclonedx-cli convert --input-file $(SBOM_FILE_NAME_CY).xml --output-file $(SBOM_FILE_NAME_CY).json
+
+sbom_fast: ## creates sbom without dependancy instalment, assumes you have cyclonedx-bom javascript package installed globally
+	mix sbom.cyclonedx -o elixir_bom.xml
+	cd assets/ && cyclonedx-bom -o ../$(SBOM_FILE_NAME_CY).xml -a ../elixir_bom.xml && cd ..
```

Comment:
@sigu won't the `SBOM_FILE_NAME_CY`  pick this file type `cyclonedx-sbom.1.0.0` that is mentioned on line 8 and we end up having something like (quaquiz.0.6.0.cyclonedx-sbom.1.0.0.json)?  or is that the naming structure we are going for?

Reply:
Yes according to the issue that is the naming convention as described https://github.com/sFractal-Podii/quizquadaminos/issues/367